inventory/management/planning/scheduled_dates.rst

```diff
@@ -18,9 +18,9 @@ suppliers, into account.
 How are Lead Times Calculated?
 ==============================
 
-.. image:: media/scheduled_dates_01.png
-    :align: center
-    :alt: Visual representation of various lead times such as customer lead time or delivery LT
+.. raw:: html
+
+      <iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQbjAf6_Py7n3bLLO0w8X7HA51ndQDn8fCWIEndg5Mt_AK9WtcaTuRM24FQ7AvIX-WS6FFQYyfjgiCI/embed?start=true&loop=false&delayms=30000" frameborder="0" width="100%" height="372px" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

Comment:
```suggestion
   <iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQbjAf6_Py7n3bLLO0w8X7HA51ndQDn8fCWIEndg5Mt_AK9WtcaTuRM24FQ7AvIX-WS6FFQYyfjgiCI/embed?start=true&loop=false&delayms=30000" frameborder="0" width="100%" height="509px" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

372 is the value we found for a Google Slide which layout has a 16:9 ratio. Here, it is different.
Considering that 640px is the default width, but the larger is 660px, I suggest to take 640px as reference. Therefore, the corresponding height is 509px (found with the "embed" tool in Google Slides).

About the identation, it's three spaces. Maybe you wrote "tabs"? But it's three spaces here.